airflow/hooks/oracle_hook.py

```diff
@@ -48,22 +48,65 @@ def get_conn(self):
               that you are connecting to (CONNECT_DATA part of TNS)
         You can set these parameters in the extra fields of your connection
         as in ``{ "dsn":"some.host.address" , "service_name":"some.service.name" }``
+        see more param detail in
+        ``https://cx-oracle.readthedocs.io/en/latest/module.html#cx_Oracle.connect``
         """
         conn = self.get_connection(self.oracle_conn_id)
+        conn_config = {
+            'user': conn.login,
```

Comment:
Maybe add a condition block to check if `login` & `password` fields are not empty and raise an error that they are required if it is empty.

Reply:
This isn't a regression or a change in behaviour, so I'll merge this.